test/checkpoint/test_checkpoint.h

```diff
@@ -55,44 +55,51 @@ typedef struct {
 } COOKIE;
 
 typedef struct {
-    char *home;                           /* Home directory */
-    const char *checkpoint_name;          /* Checkpoint name */
-    WT_CONNECTION *conn;                  /* WiredTiger connection */
-    bool debug_mode;                      /* History store stress test */
-    u_int nkeys;                          /* Keys to load */
-    u_int nops;                           /* Operations per thread */
-    FILE *logfp;                          /* Message log file. */
-    int nworkers;                         /* Number workers configured */
-    int ntables;                          /* Number tables configured */
-    int ntables_created;                  /* Number tables opened */
-    volatile int running;                 /* Whether to stop */
-    int status;                           /* Exit status */
-    bool evict_reposition_timing_stress;  /* Reposition the cursor for read operations */
-    bool sweep_stress;                    /* Sweep stress test */
-    bool failpoint_hs_delete_key_from_ts; /* Failpoint for hs key deletion. */
-    bool hs_checkpoint_timing_stress;     /* History store checkpoint timing stress */
-    bool reserved_txnid_timing_stress;    /* Reserved transaction id timing stress */
-    bool checkpoint_slow_timing_stress;   /* Checkpoint slow timing stress */
+    char *home;                  /* Home directory */
+    const char *checkpoint_name; /* Checkpoint name */
+    WT_CONNECTION *conn;         /* WiredTiger connection */
+    bool debug_mode;             /* History store stress test */
+    u_int nkeys;                 /* Keys to load */
+    u_int nops;                  /* Operations per thread */
+    FILE *logfp;                 /* Message log file. */
+    int ntables;                 /* Number tables configured */
+    int ntables_created;         /* Number tables opened */
+    int nworkers;                /* Number workers configured */
+    volatile int running;        /* Whether to stop */
```

Comment:
Make `running` a `bool`?

Reply:
I didn't really change anything here at all. The only thing I did was add a line to separate out the `bool` stress settings below which caused `clang_format` to move the comment at the end of the line inward. I'd prefer not to change `running` with this change/ticket. It seems purposely a `volatile int` and it isn't incorrect.